### PR TITLE
Add a short description to each library function (#551)

### LINKED ISSUE #551: Add a short description to each library function
On the `Cheatcodes reference` page, we list all possible cheatcodes. However, to figure out which cheatcode does what, one has to open each link and read their description. This is problematic for new users trying to figure out features of forge.

https://github.com/foundry-rs/starknet-foundry/blob/master/docs/src/appendix/forge-library.md

Add a simple description to each link on the `forge-library.md` page.

e.g. 

* `precalculate_address` - calculate an address of a contract before deployment

### PR DESCRIPTION
**Pull request by LStan**
Closes #551

## Introduced changes

- Added a short description to each library function

## Checklist

- [x] Linked relevant issue
- [x] Updated relevant documentation
- [ ] Added relevant tests
- [x] Performed self-review of the code
- [ ] Added changes to `CHANGELOG.md`

### CODE REVIEW

File: docs/src/appendix/forge-library.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-* [`declare`](forge-library/declare.md)
-* [`precalculate_address`](forge-library/precalculate_address.md)
-* [`deploy`](forge-library/deploy.md)
-* [`print`](forge-library/print.md)
-* [`read_txt`](forge-library/read_txt.md)
-* [`parse_txt`](forge-library/parse_txt.md)
-* [`read_json`](forge-library/read_json.md)
-* [`parse_json`](forge-library/parse_json.md)
+* [`declare`](forge-library/declare.md) - declares a contract and returns a struct on which `precalculate_address` and `deploy` can be called
+* [`precalculate_address`](forge-library/precalculate_address.md) - calculates a contract address that would be returned when calling `deploy`
```
Comment (piotmag769) [inline docs/src/appendix/forge-library.md:4]:
```suggestion
* [`declare`](forge-library/declare.md) - declares a contract and returns a struct on which [`precalculate_address`](forge-library/precalculate_address.md) and [`deploy`](forge-library/deploy.md) can be called
* [`precalculate_address`](forge-library/precalculate_address.md) - calculates a contract address that would be returned when calling [`deploy`](forge-library/deploy.md)
```

**Review by piotmag769 — APPROVED** [review]
Thanks for the contribution!

**Comment by piotmag769** [comment]
@LStan could you also add descriptions to `docs/src/appendix/cheatcodes.md`?

**Comment by LStan** [comment]
That issue already assigned to someone else? Is it ok if I do it?

**Comment by piotmag769** [comment]
@LStan it is not assigned to anyone, feel free to add more descriptions

**Review by Arcticae — APPROVED** [review]

**Pull request merged**